docs/app/pages/components/components-sections/panels/side-panel/side-panel.component.html

```diff
@@ -116,6 +117,9 @@ <h3>Side Panel</h3>
                     <div class="col-md-4 col-sm-12">
                         <ux-checkbox [(value)]="closeOnExternalClick">closeOnExternalClick</ux-checkbox>
                     </div>
+                    <div class="col-md-4 col-sm-12">
```

Comment:
The Bootstrap column system works on the basis of 12 columns, here you have 4 * 4 columns at the md size, so this new columns will always fold on to a new row. You could reduce them all to `col-md-3`, or just create a new row for the new checkbox.

Reply:
Ahh yes, should have considered the bootstrap grid.